docs/resources/fgs_function_event.md

```diff
@@ -0,0 +1,56 @@
+---
+subcategory: "FunctionGraph"
+---
+
+# huaweicloud_fgs_function_event
+
+Manages a test event for a specified function within HuaweiCloud.
```

Comment:
```suggestion
Manages an event for a specified function within HuaweiCloud.
```

Reply:
how about `Manages an event for testing specified function within HuaweiCloud.`